printer_mods/Long/Mantis_Dual_5015/Macros_and_Config.txt

```diff
@@ -0,0 +1,86 @@
+#Basic movements to dock and undock probe

```

Comment:
Please add a comment specifying what size printer this is for, since people with different size printers will have to adjust things in here.